crates/client/src/service.rs

```diff
@@ -464,11 +464,8 @@ pub async fn get_backend(service: &Service, backend: &Backend) -> crate::qiskit_
             crate::qiskit_target::ISAGate::I
         } else if gate == "cx" {
             crate::qiskit_target::ISAGate::CX
-        } else if gate == "rzz" {
-            // TODO: Add RZZ support when we have angle wrapping in
-            // Qiskit's target and C transpiler.
-            //            crate::qiskit_target::ISAGate::RZZ
-            continue;
+        }else if gate == "rzz" {
+            crate::qiskit_target::ISAGate::RZZ
```

Comment:
The issue with doing this now is that the transpiler will use any angle in it's output since the target can't represent the angle bounds or the angle folding/wrapping needed to restrict the angle between 0 and pi/2. That's why I left this as a TODO comment because if we added this when transpiling it would probably emit incorrect circuits. The capabilities are present in Qiskit's internal rust code, but they need to be exposed to the C API. I opened up an issue to track this here: https://github.com/Qiskit/qiskit/issues/15307